pkg/ansible/controller/controller.go

```diff
@@ -95,12 +96,16 @@ func Add(mgr manager.Manager, options Options) *controller.Controller {
 		log.Error(err, "")
 		os.Exit(1)
 	}
+
 	u := &unstructured.Unstructured{}
 	u.SetGroupVersionKind(options.GVK)
+
 	if err := c.Watch(&source.Kind{Type: u}, &crthandler.EnqueueRequestForObject{},
-		predicate.GenerationChangedPredicate{}); err != nil {
+		predicate.GenerationChangedPredicate{},
+		predicate.ResourceFilterPredicate{Selector: options.Selector}); err != nil {
```

Comment:
I'd suggest we add some vars as arguments to c.Watch() because readability of this whole line is important as well.